app/notifications/recipients/annotated_recipients.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class Recipients::AnnotatedRecipients
+  def initialize(params)
+    @params = params
+  end
+
+  def recipients; end
```

Comment:
This whole class seems redundant. We either need to properly migrate this to use the class, or just use the target_user from before (or put it in params or something).